youtube_dl/extractor/streamsb.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import random
+import string
+
+from .common import InfoExtractor
+
+
+def to_ascii_hex(str1):
+    return ''.join([format(ord(c), 'x') for c in str1])
+
+
+def generate_random_string(length):
+    return ''.join(random.choice(string.ascii_letters + string.digits) for _ in range(length))
+
+
+class StreamsbIE(InfoExtractor):
+    domain = 'viewsb.com'
+    _VALID_URL = r'https://' + domain + '/(?P<id>.+)'
+    _TEST = {
+        'url': 'https://viewsb.com/dxfvlu4qanjx',
+        'md5': '488d111a63415369bf90ea83adc8a325',
+        'info_dict': {
+            'id': 'dxfvlu4qanjx',
+            'ext': 'mp4',
+            'title': 'Sintel'
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        iframe_rel_url = self._search_regex(r'(?i)iframe src=\"(.*\.html)\"', webpage, 'iframe')
+        iframe_url = 'https://' + self.domain + iframe_rel_url
+
+        iframe_data = self._download_webpage(iframe_url, video_id)
+        app_version = self._search_regex(r'script src=".*/app\.min\.(\w+)\.js', iframe_data, 'video_code')
```

Comment:
Relax:
```suggestion
        app_version = self._search_regex(r'''<script\b[^>]+\bsrc\s*=\s*("|')(?:(?!\1).)*/app\.min\.(\d+)\.js''', iframe_data, 'app version', fatal=False) or '50'
```

Reply:
there is a problem here, the first group (the double quote) is being returned from _search_regex
I suggest the simpler:

    <script\b[^>]+\bsrc\s*=\s*["|'].*/app\.min\.(\d+)\.js